releng/test_config.yaml

```diff
@@ -251,30 +251,30 @@ k8sVersions:
     - --extract-ci-bucket=k8s-release-dev
     version: master
   beta:
+    args:
+    - --extract=ci/latest-1.28
```

Comment:
hmmm I don't think we should have 1.28 yet?

Reply:
The marker already exists in fact (https://dl.k8s.io/ci/latest-1.28.txt) and points to the latest commit in k/k at the time of writing this comment.